### ISSUE TITLE
Setup meta data for all of my.newspring.cc

### ISSUE THREAD

**Issue by jbaxleyiii** [NewSpring/Apollos#228]
@jonhorton you can speak best to what tags we need to have. Can you comment the MVP tags we need to support.

We should use react-helmet to add tags so SSR support is there.

**Comment by johnthepink** [comment]
@jonhorton did you have any thoughts on this? :heart: :heart:

**Comment by jonhorton** [comment]
How has it been a week already?! Sorry for the delay :heart: 

**Here's the start for this! Some notes:**
- I included sample content, but that should obviously be replaced with site/page specific data. 
- The og:image included is the default image if a page doesn't have one
- We'll probably need some new descriptions for this site
- If a post contains a video, I'll need to add some additional FB/Twitter tags to support that
- Rich snippet code _can_ be formatted using JSON, but we can tackle that later!
- I'll need to regenerate values for Google + Bing site verification once we're live, so it may work best to keep those commented out for now
- Off topic, but Would Google Tag Manager be supported in this build?

**Metadata**
```html
<!-- Only the fb: portion is needed here --> 
<html dir="ltr" lang="en-US" xmlns:fb="https://www.facebook.com/2008/fbml" xmlns="http://www.w3.org/1999/xhtml">

  <title>Give Now | NewSpring Church</title>
  <meta name="description" content="NewSpring Church exists to reach people far from God and teach them how to follow Jesus step by step. We currently have 11 campuses across the state of South Carolina.">

  <!--Rich Snippets-->
  <meta itemprop="name" content="Give Now | NewSpring Church">
  <meta itemprop="description" content="NewSpring Church exists to reach people far from God and teach them how to follow Jesus step by step. We currently have 11 campuses across the state of South Carolina.">
  <meta itemprop="image" content="http://ns.images.s3.amazonaws.com/newspring/site/empty-tomb.jpg">

  <!--Site Verification (Need new keys for this)-->
  <meta name="google-site-verification" content="UNk6frcn4tNFIJoQQhjFc2Rx_Pmb1SXXsyYBKUrDgXg">
  <meta name="msvalidate.01" content="611077482F129A86B86095000B59B955" />

  <!--Google+-->
  <link href="https://plus.google.com/+newspringchurch/posts" rel="author">
  <link href="https://plus.google.com/+newspringchurch" rel="publisher"/>

  <!-- Twitter Cards -->
  <meta name="twitter:title" content="Give Now | NewSpring Church">
  <meta name="twitter:description" content="NewSpring Church exists to reach people far from God and teach them how to follow Jesus step by step. We currently have 11 campuses across the state of South Carolina.">
  <meta name="twitter:image:src" content="http://ns.images.s3.amazonaws.com/newspring/site/empty-tomb.jpg">
  <meta name="twitter:site" content="@newspring">
  <meta name="twitter:creator" content="@newspring">


  <meta property="og:title" content="Welcome | NewSpring Church">
  <meta property="og:site_name" content="NewSpring Church" />
  <meta property="og:description" content="NewSpring Church exists to reach people far from God and teach them how to follow Jesus step by step. We currently have 11 campuses across the state of South Carolina.">
  <meta property="og:image" content="http://ns.images.s3.amazonaws.com/newspring/site/empty-tomb.jpg">
  <meta property="og:url" content="https://newspring.cc">
  <meta property="fb:app_id" content="167012243402263">
```

**Comment by jbaxleyiii** [comment]
@jonhorton yay! thanks so much! I'll get it added in!

**Comment by jbaxleyiii** [comment]
Fixed with #329
